conf/messages.cy

```diff
@@ -567,6 +567,7 @@ tai.incomeTax.history.endDate = Dyddiad dod i ben
 tai.incomeTax.history.taxableIncome = Incwm trethadwy
 tai.incomeTax.history.incomeTaxPaid = Treth incwm a dalwyd
 tai.incomeTax.history.taxCode = Cod treth
+tai.incomeTax.history.unavailable = xyz
```

Comment:
is that the welsh translation "xyx"- how do you pronounce it :) ?

Reply:
:-0)

Reply:
updated!